WordPress/Classes/Networking/PeopleRemote.swift

```diff
@@ -172,6 +172,80 @@ class PeopleRemote: ServiceRemoteWordPressComREST {
             failure?(error)
         })
     }
+
+
+    /// Validates Invitation Recipients.
+    ///
+    /// - Parameters:
+    ///     - siteID: The ID of the site associated.
+    ///     - usernameOrEmail: Recipient that should be validated.
+    ///     - role: Role that would be granted to the recipient.
+    ///     - completion: Closure to be executed on completion. The boolean will indicate whether the OP
+    ///       was successful, or not.
+    ///
+    func validateInvitation(siteID: Int, usernameOrEmail: String, role: Role, completion: (Bool -> ()))
+    {
+        let endpoint = "sites/\(siteID)/invites/validate"
+        let path = pathForEndpoint(endpoint, withVersion: .Version_1_1)
+
+        let parameters = [
+            "invitees"  : usernameOrEmail,
+            "role"      : role.rawValue
+        ]
+
+        wordPressComRestApi.POST(path, parameters: parameters, success: { (responseObject, httpResponse) in
+            guard let responseDict = responseObject as? [String: AnyObject],
+                    let validRecipients = responseDict["success"] as? [String] else
+            {
+                completion(false)
+                return
+            }
+
+            let success = validRecipients.contains(usernameOrEmail)
+            completion(success)
+
+        }, failure: { (error, httpResponse) in
+            completion(false)
+        })
+    }
+
+
+    /// Sends an Invitation to the specified recipient.
+    ///
+    /// - Parameters:
+    ///     - siteID: The ID of the site associated.
+    ///     - usernameOrEmail: Recipient that should receive the invite.
+    ///     - role: Role that would be granted to the recipient.
+    ///     - message: String that should be sent to the recipient.
+    ///     - completion: Closure to be executed on completion. The boolean will indicate whether the OP
+    ///       was successful, or not.
+    ///
+    func sendInvitation(siteID: Int, usernameOrEmail: String, role: Role, message: String, completion: (Bool -> ()))
+    {
+        let endpoint = "sites/\(siteID)/invites/new"
+        let path = pathForEndpoint(endpoint, withVersion: .Version_1_1)
+
+        let parameters = [
+            "invitees"  : usernameOrEmail,
+            "role"      : role.rawValue,
+            "message"   : message
+        ]
+
+        wordPressComRestApi.POST(path, parameters: parameters, success: { (responseObject, httpResponse) in
+            guard let responseDict = responseObject as? [String: AnyObject],
+                let validRecipients = responseDict["sent"] as? [String] else
+            {
+                completion(false)
```

Comment:
I'd avoid a silent failure here.  A badly written response is not the same as an invitation not being valid to the eyes of the caller of this method.